client/.eslintrc.json

```diff
@@ -4,11 +4,16 @@
         "es2021": true,
         "node": true
     },
+    "globals": {
+        "localStorage": true
+    },
     "extends": ["eslint:recommended", "airbnb"],
     "parserOptions": {
         "ecmaVersion": 12
     },
+    
     "rules": {
+        "linebreak-style": 0,
```

Comment:
what's the purpose of suppressing this rule?

Reply:
Getting lint to print expect LF linebreaks instead of CRLF , and by default all of our IDLES use LF so having this supresses that error!

Reply:
@aksh0001 can weigh in on whether we keep this rule or not

Reply:
I mean it does no harm, maybe try having windows users test the commit before this fix and see if they have the same lint issue?

Reply:
This is the issue I was referring to!
https://stackoverflow.com/questions/37826449/expected-linebreaks-to-be-lf-but-found-crlf-linebreak-style